msteams-platform/concepts/build-and-test/Shared-channels.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Shared channels
+author: Rajeshwari-v
+description:  Collaborating with shared channels.
+ms.author: surbhigupta
+localization_priority: Normal
+ms.topic: conceptual
+---
+
+# Shared channels
+
+Shared channels in Teams allow members of a channel to collaborate with users across other Teams and organizations. You can create and share a shared channel with:
```

Comment:
We ought to present the real value-add of of Shared Channels through a better 2-liner overview.